server/commands/webhook.js

```diff
@@ -0,0 +1,30 @@
+const Channel = require('../models').Channel;
+
+const slack = require('../api/slack')
+const webhookHelper = require('../helpers/webhook')
+
+const getOrCreateWebhookForChannel = async (channelId, responseUrl) => {
+  const channel = await Channel.findOne({
+    where: { channelId }
+  })
+
+  if (channel.webhookId) {
+    webhookId = channel.webhookId
```

Comment:
I'm not as familiar with Node conventions, but at least in the browser I would expect that `webhookId` should be initialized with a `let` before this, to prevent this from being a global variable / leaking outside the scope of this function.